README.md

```diff
@@ -1026,6 +1072,275 @@ export default class Page extends React.PureComponent {
+
+import { Container } from 'decentraland-ui'
+import Navbar from 'decentraland-dapps/dist/containers/Navbar'
+
+import './SomePage.css'
+
+export default class SomePage extends React.PureComponent {
+  static defaultProps = {
+    children: null
+  }
+
+  render() {
+    const { children } = this.props
+
+    return (
+      <>
+        <div className="SomePage">
+          <Container>{children}</Container>
+        </div>
+        <Navbar />
+      </>
+    )
+  }
+}
+```
+
+This `<Navbar>` will show the user's blockie and mana balance because it is connected to the store.
+
+### i18n
+
+If you are using the [Translation](https://github.com/decentraland/decentraland-dapps#translation) module, the `Navbar` contatiner comes with support for the 6 languages supported by the library.
+
+### Advanced Usage
+
+You can override any of the default translations for any locale if you need to
+
+<details><summary>Learn More</summary>
+<p>
+
+Say you want to override some translations in English, just include any or all of the following translations in your `en.json` locale file:
+
+```json
+{
+  "@dapps": {
+    "navbar": {
+      "account": {
+        "connecting": "Connecting...",
+        "signIn": "Sign In"
+      },
+      "menu": {
+        "agora": "Agora",
+        "blog": "Blog",
+        "docs": "Docs",
+        "marketplace": "Marketplace"
+      }
+    }
+  }
+}
+```
+
+## Footer
+
+The `<Footer>` container can be used in the same way as the `<Footer>` component from `decentaland-ui` but it's already connected to the redux store. You can override any `FooterProps` if you want to connect differently, and you can pass all the regular `FooterProps` to it.
+
+### Dependencies
+
+The `<Footer>` container has support for i18n out of the box if you include the [Translation](https://github.com/decentraland/decentraland-dapps#translation) module.
+
+### Usage
+
+This is an example of a `SomePage` component that uses the `<Footer>` container:
+
+```tsx
+import * as React from 'react'
+
+import { Container } from 'decentraland-ui'
+import Navbar from 'decentraland-dapps/dist/containers/Navbar'
+
+import './SomePage.css'
+
+export default class SomePage extends React.PureComponent {
+  render() {
+    const { children } = this.props
+    return (
+      <>
+        <div className="SomePage">
+          <Container>{children}</Container>
+        </div>
+        <Footer locales={['en', 'es']} />
+      </>
+    )
+  }
+}
+```
+
+This `<Footer>` will show all only English and Spanish as the options in the language dropdown. If you don't provide any it will use the 6 supported languages: `en`, `es`, `fr`, `jp`, `ko` and `zh`.
+
+### i18n
+
+If you are using the [Translation](https://github.com/decentraland/decentraland-dapps#translation) module, the `Footer` contatiner comes with support for the 6 languages supported by the library.
+
+### Advanced Usage
+
+You can override any of the default translations for any locale if you need to
```

Comment:
This is similar to the one we have in `Navbar` and other sections. What do you think of moving it to the translations section where you point the keys used?

Reply:
I know it looks repetitive but I think as the library grows it is better to keep the translation options on its respective component rather than having them all in one place, otherwise at some point it might be difficult to tell apart which translation keys belong to each component